flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/metadata/FlinkRelMdRowCount.scala

```diff
@@ -335,10 +336,19 @@ class FlinkRelMdRowCount private extends MetadataHandler[BuiltInMetadata.RowCoun
       fmq.getSelectivity(joinWithOnlyEquiPred, nonEquiPred)
     }
 
+    /** Judge */
+    val dynamicPartitionPruningFactor =
```

Comment:
add some comments to explain why we use factor here

Reply:
> add some comments to explain why we use factor here

done!